docs/Introduction/tutorials/beginners-tutorial.md

```diff
@@ -31,10 +31,23 @@ In this tutorial, you will learn about hybrid smart contracts. You will then cre
 + [1. What is a smart contract? What is a hybrid smart contract?](#1-what-is-a-smart-contract-what-is-a-hybrid-smart-contract)
 + [2. What language is a smart contract written in?](#2-what-language-is-a-smart-contract-written-in)
 + [3. What does a smart contract look like?](#3-what-does-a-smart-contract-look-like)
+  + [Define the Version of Solidity](#define-the-version-of-solidity)
```

Comment:
Did you decide that these sub-sections are necessary, or did it get pulled in from a previous branch fork?

Reply:
changed the settings for the TOC extension for the Resources page; fixing now